src/Containers.jsx

```diff
@@ -636,6 +668,20 @@ class Containers extends React.Component {
             // If there are pods to show and the generic container list is empty don't show  it at all
             if (Object.keys(partitionedContainers).length > 1 && !partitionedContainers["no-pod"].length)
                 delete partitionedContainers["no-pod"];
+
+            for (const containerid of Object.keys(this.props.containers)) {
+                const container = this.props.containers[containerid];
+                // Ignore pods and running containers
+                if ((container.State !== "exited" && container.State !== "stopped") || container.Pod)
```

Comment:
There are multiple states that mean "not running": https://github.com/cockpit-project/cockpit-podman/blob/main/src/util.js#L9 e.g. configured and created.

Reply:
Good catch.

What does Podman do in these cases? Do they just check for running and paused and exclude those?

Reply:
https://docs.podman.io/en/latest/markdown/podman-container-prune.1.html says "_stopped_ containers", but we should probably consult the actual code.

Reply:
https://github.com/containers/podman/blob/main/libpod/runtime_ctr.go#L1230-L1231:

```go
 		if state == define.ContainerStateStopped || state == define.ContainerStateExited ||
			state == define.ContainerStateCreated || state == define.ContainerStateConfigured {
```

Reply:
Answer: find containers with the status `Stopped`, `Exited`, `Created`, `Configured`

Reply:
Thanks, updated the PR.